cpp/test/sg/nvtx_test.cpp

```diff
@@ -35,7 +35,7 @@ class nvtxNextColorTest : public ::testing::Test {
     if (ML::generateNextColor(temp1) == ML::generateNextColor(temp1))
       same_string_same_color = true;
   }
-  void TearDown() {}
+
```

Comment:
hmm.. where did TearDown go? Maybe I'm reading this wrong...

Reply:
Is this cleanup required by clang-tidy? Doesn't seem worrying, just curious.

Reply:
this seems to be an issue with clang-tidy itself! Ideally it should have realised that this method is being overwritten from the base-class and thus should have not applied the tidy rules on its name. (BTW, you can see that this happens in the raft PR a lot: https://github.com/rapidsai/raft/pull/85)

And for this particular case, empty function wasn't required at all. So, just removed it.